src/common/Messaging.ts

```diff
@@ -241,13 +241,17 @@ class _Messaging {
 			response = ((await browser.runtime.sendMessage(message)) ?? null) as ReturnType<T>;
 		} catch (err) {
 			if (err instanceof Error) {
-				const messagingError = JSON.parse(err.message) as MessagingError;
-				switch (messagingError.instance) {
-					case 'Error':
-						throw new Error(messagingError.data.message);
-
-					case 'RequestError':
-						throw new RequestError(messagingError.data);
+				try {
```

Comment:
It doesn't feel right for me to have a `try...catch` inside of a `try...catch`, but it does fix the issue, so maybe we can leave it like this for now.

Reply:
Yeah, I tried finding an alternative option, but I couldn't think of a different quick fix...